src/unstract/sdk/adapters/x2text/llm_whisperer_v2/src/constants.py

```diff
@@ -46,9 +46,8 @@ class WhispererEnv:
             on failure during polling. Defaults to 5.
     """
 
-    POLL_INTERVAL = "ADAPTER_LLMW_POLL_INTERVAL"
-    MAX_POLLS = "ADAPTER_LLMW_MAX_POLLS"
-    STATUS_RETRIES = "ADAPTER_LLMW_STATUS_RETRIES"
+    WAIT_TIMEOUT = "ADAPTER_LLMW_WAIT_TIMEOUT"
+    LOG_LEVEL = "DEFAULT_LOG_LEVEL"
```

Comment:
```suggestion
    LOG_LEVEL = "LOG_LEVEL"
```
@jagadeeswaran-zipstack its better to keep the name concise - having `DEFAULT_` does not convey any extra information

Reply:
Understood, I'll make the required changes.